stl/inc/algorithm

```diff
@@ -10102,9 +10102,14 @@ namespace ranges {
             if constexpr (forward_range<_Rng> && _Prefer_iterator_copies<iterator_t<_Rng>>) {
                 const auto _Found = _RANGES _Minmax_element_unchecked(
                     _STD move(_UFirst), _STD move(_ULast), _Pass_fn(_Pred), _Pass_fn(_Proj));
+                if (_Found.min == _Found.max) {
+                    _Vty _Temp{*_Found.min};

```

Comment:
Nothing in `indirectly_copyable_storable` requires the braced form to work. Ditto below.